$MirOS: ports/www/lynx/patches/patch-src_LYMain_c,v 1.6 2007/05/20 12:06:19 tg Exp $

	cookies/bookmarks: site policy

--- src/LYMain.c.orig	Thu Aug  2 23:24:04 2007
+++ src/LYMain.c	Thu Oct 18 21:31:08 2007
@@ -65,7 +65,7 @@
 #define COOKIE_FILE "cookies"
 #define TRACE_FILE "LY-TRACE.LOG"
 #else
-#define COOKIE_FILE ".lynx_cookies"
+#define COOKIE_FILE ".etc/cookies.txt"
 #define TRACE_FILE "Lynx.trace"
 #endif /* FNAMES_8_3 */
 
@@ -1145,6 +1145,14 @@ int main(int argc,
     StrAllocCopy(LYUserAgent, LYNX_NAME);
     StrAllocCat(LYUserAgent, "/");
     StrAllocCat(LYUserAgent, LYNX_VERSION);
+#ifdef __MirBSD__
+    /*
+     * We used to add MSIE, since certain websites do not offer any
+     * functionality to Lynx users; this is bad for eg. AWStats, so
+     * a user now has to do it himself if desired; plug us instead.
+     */
+    StrAllocCat(LYUserAgent, " (compatible; MirBSD; UNIX)");
+#endif
     if (HTLibraryVersion) {
 	StrAllocCat(LYUserAgent, " libwww-FM/");
 	StrAllocCat(LYUserAgent, HTLibraryVersion);
@@ -2076,10 +2084,7 @@ int main(int argc,
      * synchronized.  - FM
      */
     if (!bookmark_page || *bookmark_page == '\0') {
-	temp = NULL;
-	HTSprintf0(&temp, "lynx_bookmarks%s", HTML_SUFFIX);
-	set_default_bookmark_page(temp);
-	FREE(temp);
+	set_default_bookmark_page("./.etc/bookmark.htm");
     }
     if (!BookmarkPage || *BookmarkPage == '\0') {
 	set_default_bookmark_page(bookmark_page);
